articles/dms/tutorial-mysql-azure-mysql-offline-portal.md

```diff
@@ -0,0 +1,254 @@
+  
+3. On the **Create Migration Service** screen, specify a name for the service, the subscription, and a new or existing resource group.
+
+4. Select a pricing tier and move to the networking screen. Offline migration capability is available in both Standard and Premium pricing tier.
+
+    For more information on costs and pricing tiers, see the [pricing page](https://aka.ms/dms-pricing).
+
+    ![Configure Azure Database Migration Service basic settings](media/tutorial-mysql-to-azure-mysql-offline-portal/05-dms-portal-create-basic.png)
+
+5. Select an existing virtual network from the list or provide the name of new virtual network to be created. Move to the review + create screen. Optionally you can add tags to the service using the tags screen.
+
+    The virtual network provides Azure Database Migration Service with access to the source SQL Server and the target Azure SQL Database instance.
+
+    ![Configure Azure Database Migration Service network settings](media/tutorial-mysql-to-azure-mysql-offline-portal/06-dms-portal-create-networking.png)
+
+    For more information about how to create a virtual network in the Azure portal, see the article [Create a virtual network using the Azure portal](../virtual-network/quick-create-portal.md).
+
+6. Review the configurations and select **Create** to create the service.
+    
+    ![Azure Database Migration Service create](media/tutorial-mysql-to-azure-mysql-offline-portal/07-dms-portal-create-submit.png)
+
+## Create a migration project
+
+After the service is created, locate it within the Azure portal, open it, and then create a new migration project.  
+
+1. In the Azure portal, select **All services**, search for Azure Database Migration Service, and then select **Azure Database Migration Services**.
+
+    ![Locate all instances of Azure Database Migration Service](media/tutorial-mysql-to-azure-mysql-offline-portal/08-01-dms-portal-searchservice.png)
+
+2. Select your migration service instance from the search results and select + **New Migration Project**.
+    
+    ![Create a new migration project](media/tutorial-mysql-to-azure-mysql-offline-portal/08-02-dms-portal-newproject.png)
+
+3. On the **New migration project** screen, specify a name for the project, in the **Source server type** selection box, select **MySQL**, in the **Target server type** selection box, select **Azure Database For MySQL** and in the **Migration activity type** selection box, select **Data migration**
+
+    ![Create Database Migration Service Project](media/tutorial-mysql-to-azure-mysql-offline-portal/09-dms-portal-project-mysql-create.png)
+
+    > [!NOTE]
+    > Alternately, you can choose **Create project only** to create the migration project now and execute the migration later.
+
+## Configure migration project
+
+1. On the **Select source** screen, specify the connection details for the source MySQL instance, and select **Next : Select target>>**
```

Comment:
Should we talk about the read-only option on this screen, and make sure it calls out that it sets the entire server to read-only, not just the selected databases.

Reply:
Done